app/src/main/java/com/gathering/android/di/AuthModule.kt

```diff
@@ -54,9 +53,10 @@ class AuthModule {
     @Provides
     @Singleton
     fun provideSignInRepository(
```

Comment:
do we need this provider method ? the class has an @inject itself. we should be able to move the @Singleton there and get rid of this.